src/module.h

```diff
@@ -166,6 +166,10 @@ class Module : public Package
 
     Symbol *sfilename;          // symbol for filename
 
+    Symbol *massert;            // module assert function
+    Symbol *munittest;          // module unittest failure function
+    Symbol *marray;             // module array bounds function
```

Comment:
I would prefer that these weren't re-added.  I don't think the changes to the backend to support these are _that_ big.

Reply:
They won't be, this is a WIP pull.